src/components/Wizard/Patterns/WizardPatternBody.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { noop, EmptyState, Spinner, Wizard } from '../../../index';
+
+/**
+ * WizardPatternBody - the Wizard Pattern Body component.
+ */
+const WizardPatternBody = ({
+  loading,
+  steps,
+  activeStepIndex,
+  goToStep,
+  loadingTitle,
+  loadingMessage,
+  activeStepStr
+}) => {
+  if (loading) {
+    return (
+      <Wizard.Row>
+        <Wizard.Main>
+          <EmptyState>
+            <Spinner size="lg" className="blank-slate-pf-icon" loading />
+            <EmptyState.Action>
+              <h3>{loadingTitle}</h3>
+            </EmptyState.Action>
+            <EmptyState.Action secondary>
+              <p>{loadingMessage}</p>
+            </EmptyState.Action>
+          </EmptyState>
+        </Wizard.Main>
+      </Wizard.Row>
+    );
+  }
+
+  const stepProps = (stepIndex, title) => {
+    const label = (stepIndex + 1).toString();
+    return {
+      key: `wizard-step-${title}`,
+      stepIndex,
+      label,
+      step: label,
+      title,
+      activeStep: activeStepStr
+    };
+  };
+
+  const step = steps[activeStepIndex];
+
+  const renderedStep =
+    step && step.render && step.render(activeStepIndex, step.title);
+
+  return (
+    <React.Fragment>
+      <Wizard.Steps
+        steps={steps.map((stepObj, index) => (
+          <Wizard.Step
+            {...stepProps(index, stepObj.title)}
+            onClick={() => goToStep(index)}
```

Comment:
Each render of every `<Wizard.Step>` will have new `onClick` prop value, I know it's not a big deal, but just wondering what's the ideal way to handle this?

@priley86 does it make sense to use `ref` and bind the method once in parent/owner component's (`WizardPatternBody`) constructor?

Reply:
good point, we haven't done this much to date in this repo... but i wouldn't be opposed to this where it makes sense. maybe you can comb back through this in a future PR with more suggestions/tweaks on your side... that would be really nice 👍. I think we are going to quickly need to wrap up this iteration soon... but fine w/ follow up PRs at any point...

Reply:
Agreed. I think this is something that is negligible enough for now that we should come back to it later.